prime-router/docker-compose.yml

```diff
@@ -66,6 +66,17 @@ services:
     networks:
       - prime-router_build
 
+  # a SOAP endpoint to test against
+  soap-webservice:
+    image: castlemock/castlemock
+    ports:
+      - "8087:8080"
+    volumes:
+      - ../.environment/soap_service:/root/.castlemock
+      - ../.environment/soap_service:/castlemock/castlemock
+    networks:
+      - prime-router_build
```

Comment:
This is all you get for setting up CastleMock in Docker. It's very sparse. That's why I have the other files up above. There's also very little documentation. 🤷‍♂️